tensornetwork/block_sparse/charge.py

```diff
@@ -223,22 +221,28 @@ def __add__(self, other: "BaseCharge") -> "BaseCharge":
     Returns:
       BaseCharge: The result of fusing `self` with `other`.
     """
+    if len(self.charge_labels) == 0 or len(other.charge_labels) == 0:
+      obj = self.__new__(type(self))
+      obj.__init__(
+          np.empty((0, self.num_symmetries), dtype=self.dtype),
```

Comment:
What's the point of having an array with a dimension size of 0?

Reply:
For consistency the class needs to be initialized